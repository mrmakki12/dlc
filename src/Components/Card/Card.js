import React from "react";

export const Card = ({data}) => {
    return (
        <div className="col" data-testid='card'>
            <div className="card text-dark">
                <img className="card-img-top" src={data.img} />
                <div className="card-body">
                    <h4 className="card-title">{data.title}</h4>
                    <p className="card-text">{data.description}</p>
                    <b><p className="card-text">${data.price}</p></b>
                </div>
            </div>
        </div>
    )
}