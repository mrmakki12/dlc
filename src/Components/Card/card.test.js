import React from "react";
import { Card } from './Card.js';
import { cleanup, render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

afterEach(cleanup);

// basic render
test("render Card without crashing", () => {
    render(<Card />)
});