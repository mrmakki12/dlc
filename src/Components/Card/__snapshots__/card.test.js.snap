// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="col"
  data-testid="card"
>
  <div
    className="card text-white bg-dark"
  >
    <img
      className="card-img-top"
      src="/pictures/background.jpg"
    />
    <div
      className="card-body"
    >
      <h4
        className="card-title"
      >
        Hiking Simulator: Jungle Adventure
      </h4>
      <b>
        <p
          className="card-text"
        >
          $
          free 99
        </p>
      </b>
    </div>
  </div>
</div>
`;
